content/en/docs/tasks/administer-cluster/declare-network-policy.md

```diff
@@ -49,7 +50,7 @@ kubectl expose deployment nginx --port=80
 service/nginx exposed
 ```
 
-This runs a `nginx` pods in the default namespace, and exposes it through a service called `nginx`.
+The above commands create a Deployment with an nginx Pod and expose the Deployment through a Service named `nginx`. The `nginx` Pod, and the Deployment that owns it, are in the `default` namespace.
```

Comment:
Line 53, possible edit:
Replace line 53 with:
The `nginx` Pod and Deployment are found in the `default` namespace.